src/settings.py

```diff
@@ -325,6 +326,7 @@
 PING_IF_NUMS_ACCS = {}
 
 is_role = lambda plyr, rol: rol in ROLES and plyr in ROLES[rol]
+start_votes_required = lambda num_players: min((num_players + 1) / 2.2, 4)
```

Comment:
It was originally all uppercase, but I turned it lowercase after some of the other devs told me to. I can change it again of course, though the other lambdas in settings.py are also lowercase.

Regarding turning it into a variable, I'm not so sure. I guess I could turn it into a simple variable of a percentage of users, but that wouldn't allow more "complex" calculations like the current one and requires a more akward syntax to completely disable it (I guess you could set it to 0%).